src/viking.js

```diff
@@ -1,11 +1,124 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function() {
+  return this.strength;
+};
+
+Soldier.prototype.receiveDamage = function (damage) {
+  this.health -= damage;
+}
 
 // Viking
-function Viking() {}
+
+Viking.prototype = Object.create(Soldier.prototype);
+
+function Viking(name, health, strength) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+}
+
+Viking.prototype.attack = function() {
+  return this.strength;
+}
+
+Viking.prototype.receiveDamage = function(damage) {
+  this.health -= damage;
+  if(this.health > 0) {
+    return `${this.name} has received ${damage} points of damage`;
+  } else {
+    return `${this.name} has died in act of combat`;
+  }
+}
+
+Viking.prototype.battleCry = function() {
+  return `Odin Owns You All!`;
+}
 
 // Saxon
-function Saxon() {}
+
+Saxon.prototype = Object.create(Soldier.prototype);
+
+function Saxon(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Saxon.prototype.attack = function() {
+  return this.strength;
+}
+
+Saxon.prototype.receiveDamage = function(damage) {
+  this.health -= damage;
+  if(this.health > 0) {
+    return `A Saxon has received ${damage} points of damage`;
+  } else {
+    return `A Saxon has died in combat`;
+  }
+}
 
 // War
-function War() {}
+
+function War() {
+  this.vikingArmy = [];
+  this.saxonArmy = [];
+}
+
+// QUESTION - do we have to use the new Viking / new Saxon constructor when we add to the vikingArmy / saxonArmy array?
```

Comment:
The test does that for you: 
Line 230 in the VikingSpec.js inside tests folder :)
But if you haven't created the Viking or Saxon class it will break